package.json

```diff
@@ -6,19 +6,19 @@
   "private": true,
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "start": "npm run dev",
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run",
+    "server": "node server.js",
+    "start": "npm run build && node server.js",
```

Comment:
I'm using the 'postinstall' keyword in my package.json, not that it matters, if you find it useful - https://github.com/Volunteero/volunteero-front-end-base/blob/master/package.json (just realized that I have some repetitive code there)

Reply:
My other comment didn't show up but it was about the server.js and that I have almost the same for the angular https://github.com/Volunteero/volunteero-front-end-base/blob/master/server.js. Looks good

Reply:
Awesome! Thanks a lot dude, `postinstall` is indeed a nice idea! 
I will also take a look at how you handle dev dependencies in your app. 
This one ended up with a 85 MB package and I am not really happy with it - maybe any tips?

Thanks @dmarkov00  🤙